README.md

```diff
@@ -1,22 +1,22 @@
 [![cmake workflow](https://github.com/geoscienceaustralia/DynAdjust/actions/workflows/cmake_release.yml/badge.svg)](https://github.com/geoscienceaustralia/DynAdjust/actions/workflows/cmake_release.yml)
 [![cmake tests](https://github.com/geoscienceaustralia/DynAdjust/actions/workflows/test_coverage.yml/badge.svg)](https://github.com/geoscienceaustralia/DynAdjust/actions/workflows/test_coverage.yml)
-[![docker build](https://github.com/icsm-au/DynAdjust/actions/workflows/docker.yml/badge.svg)](https://github.com/icsm-au/DynAdjust/actions/workflows/docker.yml)
-[![codacy badge](https://img.shields.io/codacy/grade/a3944cda0c72445f8a13b1f82b64f714)](https://app.codacy.com/gh/icsm-au/DynAdjust/dashboard)
+[![docker build](https://github.com/GeoscienceAustralia/DynAdjust/actions/workflows/docker.yml/badge.svg)](https://github.com/GeoscienceAustralia/DynAdjust/actions/workflows/docker.yml)
+[![codacy badge](https://img.shields.io/codacy/grade/a3944cda0c72445f8a13b1f82b64f714)](https://app.codacy.com/gh/GeoscienceAustralia/DynAdjust/dashboard)
 [![cii best practices](https://img.shields.io/badge/cii%20best%20practices-passing-success)](https://bestpractices.coreinfrastructure.org/projects/4894)
 [![Coverage Status](https://coveralls.io/repos/github/GeoscienceAustralia/DynAdjust/badge.svg)](https://coveralls.io/github/GeoscienceAustralia/DynAdjust)
-[![codecov status](https://img.shields.io/codecov/c/github/icsm-au/dynadjust)](https://codecov.io/gh/icsm-au/DynAdjust)
+[![codecov status](https://img.shields.io/codecov/c/github/GeoscienceAustralia/dynadjust)](https://codecov.io/gh/GeoscienceAustralia/DynAdjust)
```

Comment:
Since GA doesn't have an account with codecov yet, this link (https://codecov.io/gh/GeoscienceAustralia/DynAdjust) will be broken.  That's why this badge still points to icsm (https://codecov.io/gh/icsm-au/DynAdjust). I suggest leave as icsm-au for now.